scripts/cnv_cromwell_tests/somatic/cnv_somatic_pair_wes_tumor-only_workflow.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Can we chat in person to clarify the differences between case, sample, pair, etc.?  Also note that I called the copy-ratio workflow a "bam_workflow", since it can be run on a single normal or tumor bam, but I don't know if this is the appropriate term either.